### DISCUSSION TITLE
PandasAI in production

### DISCUSSION THREAD

**Discussion by fletchsims** [sinaptik-ai/pandas-ai#1139] [General]
Hi all, I'm wondering if anyone has experience or experimented with deploying PandasAI in a production environment with multiple users? I'm kind of curious about the steps at a high level you went to deploying this type of service. Was it done via a REST API, did you use the Agent feature of PandasAI, etc. Thank you in advance!

**Comment by caniko** [comment]
They have a very useless license. You have to pay if you want to use anything under pandasai/ee. Spoiler alert, a lot of the useful code is in there.
